### ISSUE TITLE
Workaround for validating amp html pages on yahoo small business hosting

### ISSUE THREAD

**Issue by the-mj** [ampproject/amphtml#13488]
I am absolutely frustrated with validating and indexing amp html pages for my client. To make matters worse the website is hosted at Yahoo small business hosting. The amp validator validates my amp html page but google refuses to index it stating the "user authored java script" error. After 2 agonizing hours i finally found the culprit. Yahoo injects a lexity tracking script (by default) on every page hosted on their platform.
I think just because the script is injected after the closing html tag, amp validator validates the page as is.
When googlebot crawls the amp page, it shows the error.
After a long time i decided to give amp html a shot but this error disappoints me but my client is not in a position to change the web host. (Her plan ends next year in July) 
Can anyone show me a workaround on how to get the pages indexed?

**Comment by choumx** [comment]
@the-mj Sorry to hear you're having trouble. This sounds like something to raise with Yahoo -- have you tried engaging their support channels?

**Comment by the-mj** [comment]
@choumx thank you for replying. Yes, I have contacted support and they are unhelpful. I found a similar thread in their forums which seems to be the final nail in the coffin. Link for you to check: https://help.smallbusiness.yahoo.net/s/question/0D51a00001CE5vLCAT/how-to-remove-the-lexity-script-i-am-trying-to-run-an-amp-page-but-the-script-shows-up-leading-to-validation-failure

I know there's nothing wrong with amp code but the annoying script injection leads to instant invalid amp pages. 

Is there a way that amp can allow scripts from np.lexity.com

This is the script //np.lexity.com/embed/YW/ff6d67d55198e1a818b5ec01dc6c5c53?id=fd93d11e7edb

I know this is too much too ask but my client is not in a position to change the host atleast for a year. The amount pages was probably their last shot to get get onto mobile Google seeps.
